index.coffee

```diff
@@ -96,6 +96,12 @@ export shouldOpenInNewWindow = (url) ->
 		return false if urlObj.href.match urlRegex
 	return true
 
+# add trailing slash 
+export addTrailingSlash = (to) -> 
+	url = new URL to
+	url.pathname += if url.pathname.endsWith "/" then "" else "/"
+	url.toString()
```

Comment:
How about this instead.  It maybe computationally cheaper (don't need to construct a `URL` instance) and is more backwards compatible (since `endsWidth` isn't available in IE11).

```suggestion
	if to.match /\/$/ then to else to + '/'
```

Reply:
replacing the URL object approach with a direct string manipulation had some problems when dealing with urls such as "https://www.happyfamilyorganics.com/learning-center/all?postCategory=postnatal". Therefore I dropped the `endsWith` method for the regex match, but kept the URL construct.